pkg/retry/config.go

```diff
@@ -25,12 +25,46 @@ import (
 // Config defines the set of environment variables required
 // for running the retry service.
 type Config struct {
-	Port string `env:"PORT,default=8081"`
+	AppID              string `env:"GITHUB_APP_ID,required"`
+	BigQueryID         string `env:"BIG_QUERY_ID,required"`
+	BucketURL          string `env:"BUCKET_URL,required"`
+	LockTTLClockSkewMS int    `env:"LOCK_TTL_CLOCK_SKEW_MS,required"`
+	LockTTLMinutes     int    `env:"LOCK_TTL_MINUTES,required"`
+	ProjectID          string `env:"PROJECT_ID,required"`
+	Port               string `env:"PORT,default=8081"`
```

Comment:
Any reason why we don't want 8080 to be the default port

Reply:
The webhook service defaults to that port.